plugins/mcreator-themes/themes/default_dark/images/img_editor/stamp.svg

```diff
@@ -3,15 +3,15 @@
 <svg width="20" height="20" version="1.1" viewBox="0 0 5.2917 5.2917" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
     <defs>
         <mask id="mask-powermask-path-effect59" maskUnits="userSpaceOnUse">
-            <path id="mask-powermask-path-effect59_box" d="m0.05834 1.9104h5.175v3.4552h-5.175z" fill="#fff"/>
+            <path id="mask-powermask-path-effect59_box" d="m0.05834 1.9104h5.175v3.4552h-5.175z" fill="#231F20"/>
```

Comment:
The colors applied to mask components are used to determine their transparency, so this value should be reverted.

Reply:
Tried that but now the curved part is black in dark mode

Reply:
Uh-huh, it's now getting filled with the right source color, but nothing more :thinking:
Can we do the same without a mask?

Reply:
Idea renders it correctly, but not the JSVG with your suggestion

Reply:
I tried to remove the mask from the icon, here are the commands of the path that looked close to the mask:
`m2.9083 2.8104 q0.075 1.375 1.425 1.425v0.1h-3.375v-0.1q1.35 -0.05 1.425 -1.425z`
Path like this would also eliminate the need for the rectange between the mask and the ellipse.